Removing videos and replacing with screenshots and written directions (#1938)
fishtown-analytics/docs.getdbt.com: website/docs/docs/dbt-cloud/access-control/enterprise-permissions.md

## How to Set Up RBAC Groups in dbt Cloud

Role-Based Access Control (RBAC) is helpful for automatically assigning permissions to dbt admins based on their SSO provider group associations.

- **If you are on a dbt Labs Hosted dbt Cloud instance:**
Contact support via the webapp button or support@getdbt.com to turn on this feature.
- **If you are on a customer deployed dbt Cloud instance:**
Contact your account manager for instructions on how to turn on this feature.

> Open the **Main Menu** and click **Account Settings**. From the **Team** section, click **Groups**

Reviewer: ```suggestion
Click the gear icon to the top right and select **Account Settings**. From the **Team** section, click **Groups**
```

<Lightbox src="/img/docs/dbt-cloud/Select-Groups-RBAC.png" title="Navigate to Groups"/>

> Select an existing group or create a new group to add RBAC. Name the group (this can be any name you like, but it's recommended to keep it consistent with the SSO groups), and then configure the SSO provider groups you want to add RBAC (these fields are case sensitive and must match the source group formatting) by clicking **Add** in the **SSO Mapping** section.

Reviewer: ```suggestion
Select an existing group or create a new group to add RBAC. Name the group (this can be any name you like, but it's recommended to keep it consistent with the SSO groups), and then configure the SSO provider groups you want to add RBAC (these fields are case sensitive and must match the source group formatting) by clicking **Add** in the **SSO** section.
```

Next, configure the permissions for users within those groups by clicking **Add** in the **Access** section of the window.

<Lightbox src="/img/docs/dbt-cloud/Configure-SSO-Access.png" title="Configure SSO groups and Access permissions"/>

> When you've completed your configurations, scroll up and click **Save** in the top right-hand corner. Users will begin to populate the group automatically once they have signed in to dbt Cloud with their SSO credentials.

Reviewer: ```suggestion
When you've completed your configurations, click **Save**. Users will begin to populate the group automatically once they have signed in to dbt Cloud with their SSO credentials.
```

> <Lightbox src="/img/docs/dbt-cloud/Save-Group-Information.png" title="Save group configuration"/>

Reviewer: I don't think we need this. It looks like you only click Save once now!
```suggestion
```
